Bump `open-aea` and `open-autonomy` frameworks (#57)
valory-xyz/price-oracle: docs/index.md

To understand the deployment process better, follow the deployment guide [here](https://docs.autonolas.network/open-autonomy/guides/deploy_service/).

## Querying the service

Querying autonomous services can become very simple by using the 
[Open Autonomy Client SDK](https://github.com/valory-xyz/open-autonomy-client).

> This is a library that helps to query multi-agent systems built with the open-autonomy framework. 

Reviewer: ```suggestion
This is a library that helps to query multi-agent systems built with the Open Autonomy framework. 
```

> It allows one to simply perform a request to a service as if it were a single endpoint. 

Author: I do not understand this suggestion.

> The SDK in the background requests the information from multiple agents in order to return a result for which 
> it is safe to assume that the agents have reached consensus on.

Reviewer: ```suggestion
The SDK queries multiple agents in the background to retrieve information and returns a result that is presumed to be reached by consensus among the agents.
```

Let's take a look at a simple example, using the SDK. First of all we need to make sure 
that we have the necessary requirements installed:

```bash
pip install open-autonomy-client
pip install aiohttp
```

Having installed the requirements and while running the hardhat demo above, you can use this simple script 
to get a result for which the agents have reached consensus on:

```python
import asyncio
import json
from open_autonomy_client.client import Client

PUB_KEYS_LIST = ['0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266', '0x70997970C51812dc3A010C7d01b50e0d17dc79C8',
                 '0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC', '0x90F79bf6EB2c4f870365E785982E1f101E93b906']
AGENTS_URLS_LIST = [
    f"http://127.0.0.1:{i}"
    for i in range(8000, 8000 + len(PUB_KEYS_LIST))
]

async def fetch():
    client = Client(urls=AGENTS_URLS_LIST, keys=PUB_KEYS_LIST)
    agents_data = await client.fetch()
    print(json.dumps(agents_data, indent=4))

if __name__ == '__main__':
  asyncio.run(fetch())
```

> Let's take a look at this script step by step:

Reviewer: ```suggestion
Let's take a look at this script step by step:

```
